### ISSUE TITLE
Update GraphCast scripts from DeepMind repository

### ISSUE THREAD

**Issue by SadeghTabas-NOAA** [NOAA-EMC/graphcast#14]
The GraphCast scripts on DeepMind's repository got significant updates that should be applied to the EMC GraphCast repository. 
Updates include:

- Updating Readme
- Dependencies version update
- updates on data_utils.py and data_utils_test.py
- updates on casting.py
- updates on the normalization.py script
- updates on the rollout.py script
- updates on xarray_jax.py
- updates on graphcast_demo.ipynb
- added scripts for calculation of top of atmosphere solar radiation in case of operational use
- some other minor scripts updates
